src/cli/TextStream.cpp

```diff
@@ -58,6 +58,11 @@ TextStream::TextStream(const QByteArray& array, QIODevice::OpenMode openMode)
     detectCodec();
 }
 
+void TextStream::write(const char* str)
+{
+    *this << QString(str);
```

Comment:
Add a comment above this line explaining that this is a workaround for a Qt issue that is resolved in Qt 6.x.